Failures while building `vstest` tests in VMR

Here are the errors encountered when we tried to enable building tests in VMR:

```
C:\git\dotnet\src\vstest\test\Microsoft.TestPlatform.Acceptance.IntegrationTests\TranslationLayerTests\EventHandler\DiscoveryEventHandler.cs(153,10): error IDE0055: Fix formatting (https://learn.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0055) [C:\git\dotnet\src\vstest\test\Microsoft.TestPlatform.Acceptance.IntegrationTests\Microsoft.TestPlatform.Acceptance.IntegrationTests.csproj::TargetFramework=net9.0]
C:\git\dotnet\src\vstest\test\Microsoft.TestPlatform.Acceptance.IntegrationTests\TranslationLayerTests\EventHandler\DiscoveryEventHandler.cs(153,10): error IDE0055: Fix formatting (https://learn.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0055) [C:\git\dotnet\src\vstest\test\Microsoft.TestPlatform.Acceptance.IntegrationTests\Microsoft.TestPlatform.Acceptance.IntegrationTests.csproj::TargetFramework=net48]
```

For repro, add `/p:DotNetBuildTests=true` to your Windows VMR build command, i.e.: `build -prepareMachine -c Release /p:TargetOS=windows /p:TargetArchitecture=x64 /bl /p:DotNetBuildTests=true`

--- a/test/Microsoft.TestPlatform.Acceptance.IntegrationTests/TranslationLayerTests/EventHandler/DiscoveryEventHandler.cs
+++ b/test/Microsoft.TestPlatform.Acceptance.IntegrationTests/TranslationLayerTests/EventHandler/DiscoveryEventHandler.cs
@@ -150,7 +150,7 @@ public class DiscoveryEventHandlerForBatchSize : ITestDiscoveryEventsHandler2, I
         if (level == TestMessageLevel.Error)
         {
             Console.WriteLine($"ERROR:{message}");
-        };
+        }
     }
 
     public void HandleDiscoveryComplete(DiscoveryCompleteEventArgs discoveryCompleteEventArgs, IEnumerable<TestCase>? lastChunk)

